extensions/wikia/PortableInfobox/styles/PortableInfobox.scss

```diff
@@ -79,9 +79,31 @@
 }
 
 .pi-smart-data {
+	border-left-style: solid;
+	border-left-width: 1px;
 	display: block;
 	flex-direction: column;
 	text-align: center;
+
+	.pi-data-value {
+		padding: 5px 10px 0;
```

Comment:
are these values used only here? Maybe it is worth to extract them to variables?